### ISSUE TITLE
How to smooth 2d position？

### ISSUE THREAD

**Issue by zhouhang95** [Shimingyi/MotioNet#6]
Now we haven't apply any smoothing process on the 2d input and output, you can do it by yourself to get better results like we show in the video, but here we want to show the original production.

Could you give me a concrete algorithm？

**Comment by Shimingyi** [comment]
It can be done by different ways, like pose tracking with [PoseFlow](https://github.com/YuliangXiu/PoseFlow) or interpolation module in scipy. I will release a version with [splev](https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.splev.html#scipy.interpolate.splev) smoothing in weekend.

**Comment by CHDQ** [comment]
@Shimingyi 同问，2d pose平滑有更新了么

**Comment by Shimingyi** [comment]
@CHDQ No... But I will suggest to run this [branch ](https://github.com/soulslicer/STAF/tree/staf)of openpose to get better input firstly.
